.github/workflows/build-and-install.yml

```diff
@@ -124,3 +124,80 @@ jobs:
         run: |
           docker run -w /netdata test \
               /bin/sh -c '/netdata/rmjsonc.sh && ./netdata-installer.sh --dont-wait --dont-start-it --require-cloud'
+  aws-kinesis-build:
+    name: With AWS Kinesis SDK
+    strategy:
+      fail-fast: false
+      matrix:
+        distro:
+          # XXX: CentOS 8.x is broken (libjudy)
+          # - 'centos:8'
+          - 'debian:buster'
+          - 'fedora:32'
+          - 'ubuntu:20.04'
+        include:
+          #- distro: 'centos:8'
+          #  pre: >-
+          #    yum -y update &&
+          #    yum -y groupinstall 'Development Tools' &&
+          #    yum -y install libcurl-devel openssl-devel libuuid-devel
+          #  build_kinesis: >-
+          #    git clone https://github.com/aws/aws-sdk-cpp.git &&
+          #    cmake -DCMAKE_INSTALL_LIBDIR=/usr/lib
+          #    -DCMAKE_INSTALL_INCLUDEDIR=/usr/include
+          #    -DBUILD_SHARED_LIBS=OFF -DBUILD_ONLY=kinesis
+          #    ./aws-sdk-cpp
+          - distro: 'debian:buster'
+            pre: >-
+              apt-get update &&
+              apt-get install -y build-essential &&
+              apt-get install -y libcurl4-openssl-dev libssl-dev uuid-dev zlib1g-dev libpulse-dev
+            build_kinesis: >-
+              git clone https://github.com/aws/aws-sdk-cpp.git &&
+              cmake -DCMAKE_INSTALL_LIBDIR=/usr/lib
+              -DCMAKE_INSTALL_INCLUDEDIR=/usr/include
+              -DBUILD_SHARED_LIBS=OFF -DBUILD_ONLY=kinesis
+              ./aws-sdk-cpp
+          - distro: 'fedora:32'
+            pre: >-
+              dnf -y update &&
+              dnf -y groupinstall 'Development Tools' &&
+              dnf -y install libcurl-devel openssl-devel libuuid-devel
+            build_kinesis: >-
+              git clone https://github.com/aws/aws-sdk-cpp.git &&
+              cmake -DCMAKE_INSTALL_LIBDIR=/usr/lib
+              -DCMAKE_INSTALL_INCLUDEDIR=/usr/include
+              -DBUILD_SHARED_LIBS=OFF -DBUILD_ONLY=kinesis
+              ./aws-sdk-cpp
```

Comment:
It won't install the AWS SDK libraries and its third-party dependencies. I think it should be
```
cmake -DCMAKE_INSTALL_PREFIX=/usr -DBUILD_ONLY=kinesis ./aws-sdk-cpp &&
make &&
make install
```

Reply:
Okay! I'll update this thanks!

Reply:
Of course d'oh I almost forgot that `cmake` doesn't actually build anything! 🤣 Whoops!